addons/website/views/snippets/s_references_social.xml

```diff
@@ -0,0 +1,93 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_references_social" name="References Social">
+    <section class="s_references_social o_cc o_cc1 pt64 pb64">
+        <div class="container">
+            <div class="row">
+                <div class="col-lg-12">
+                    <h2 style="text-align: center;">Our valued partners</h2>
+                    <p class="lead" style="text-align: center;">We are in good company.</p>
+                </div>
+                <div class="col-lg-3 pt16 pb16">
+                    <p style="text-align: center;">
+                        <img class="img img-fluid" src="/web/image/website.s_reference_demo_image_1" alt=""/>
+                    </p>
+                    <h3 class="h5-fs" style="text-align: center;">Amsterdam</h3>
```

Comment:
```suggestion
                    <img class="img img-fluid" src="/web/image/website.s_reference_demo_image_1" alt=""/>
                    <h3 class="h5-fs" style="text-align: center;"><br/>Amsterdam</h3>
```

I actually think it's better to do it like this. When the image is wrapped in a `<p>` and I try to delete the empty line (hitting backspace), the heading becomes a paragraph